packages/client/src/__tests__/integration/happy/queryTimeout/test.ts

```diff
@@ -0,0 +1,120 @@
+import { ClientEngineType } from '@prisma/sdk'
+import fs from 'fs'
+import os from 'os'
+import path from 'path'
+
+import { generateTestClient } from '../../../../utils/getTestClient'
+
+const buildSchema = (engineType?: string) => `
```

Comment:
```suggestion
const buildSchema = (engineType?: string) => /* Prisma */ `
```
Small trick of our VS Code syntax highlighting